src/wasm-traversal.h

```diff
@@ -247,7 +247,8 @@ struct Walker : public VisitorType {
       self->visitExport(curr.get());
     }
     for (auto& curr : module->globals) {
-      self->walkGlobal(curr.get());
+      // TODO: Stop parsing from inserting nullptr into globals
```

Comment:
After parsing test/spec/globals.wast we're left will a nullptr in the list of globals. This causes wasm2asm to crash (and is clearly incorrect) without this check.